scripts/DSRA_outputs2postgres_lfs.py

```diff
@@ -146,11 +146,16 @@ def GetDataframeForScenario(url, repo_list, retrofitPrefix, eqscenario, columnCo
     #dfLosses['sl_BldgT'] = dfLosses['sl_Str'] + dfLosses['sl_NStr'] + dfLosses['sl_Cont']  
     dfLosses = dfLosses.add_suffix("_{}".format(retrofitPrefix))
     dfLosses.rename(columns={"AssetID_{}".format(retrofitPrefix):"AssetID"}, inplace=True)
+
+    eq_url = f'https://raw.githubusercontent.com/OpenDRR/earthquake-scenarios/master/initializations/s_Hazard_{eqscenario}.ini'
+    response = requests.get(eq_url, headers={'Authorization': 'token {}'.format(auth.get('auth', 'github_token'))}).text
```

Comment:
@drotheram @wkhchow since the repo is open now we probably don't need this. As well, we might want to think about downloading required files using a release tag to ensure the build is working off of a stable release. Just an idea.

Reply:
Building off the release should definitely be the next change. I would like to refactor this whole script to pull the CSVs into the filesystem to align with the rest of the stack. This script is too memory intensive